<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClangTools::Internal::RunSettingsWidget</class>
 <widget class="QWidget" name="ClangTools::Internal::RunSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Run Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="CppTools::ClangDiagnosticConfigsSelectionWidget" name="diagnosticWidget" native="true"/>
      </item>
      <item>
       <widget class="QCheckBox" name="buildBeforeAnalysis">
        <property name="text">
         <string>Build the project before analysis</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="processesLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Parallel jobs:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="parallelJobsSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CppTools::ClangDiagnosticConfigsSelectionWidget</class>
   <extends>QWidget</extends>
   <header>cpptools/clangdiagnosticconfigsselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
